src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -57,22 +63,22 @@ internal sealed class SourceConstructorSymbol : SourceConstructorSymbolBase
                     diagnostics.Add(ErrorCode.ERR_ExternHasConstructorInitializer, location, this);
                 }
 
-                if (hasBody)
+                if (hasAnyBody)
```

Comment:
different code used hasBody to mean different things.  Some code uses it to mean "has a block body" other code uses it to mean "has a block or expression body".  I've introduced a helper and have made this consistent.  Now it's "hasBlockBody" and "hasAnyBody" to make it clear across all symbols.

Reply:
this was done as it actually caught bugs as i was doing this work.